server/app.py

```diff
@@ -50,7 +50,9 @@
 api.add_resource(ProspectsResources.InheritPreviousStepProspects, '/steps/prospects')
 api.add_resource(CampaignResources.GetCampaign, '/campaigns/<int:id>')
 api.add_resource(CampaignResources.CampaignProspects, '/campaign/<int:id>/prospects')
-api.add_resource(CampaignResources.CreateStepToCampaign, '/campaign/<int:id>/steps')
 api.add_resource(StepResources.ExecuteStep, '/gmail/send')
+api.add_resource(StepResources.CreateStep, '/campaigns/<int:id>/steps', endpoint='steps')
+api.add_resource(StepResources.UpdateStep, '/steps/<int:id>', endpoint='step')

```

Comment:
Then at the same time, we can modify routes to just /campaigns/steps/<int:id>

Reply:
I am not sure if that will work with the POST route, b/c it doesn't require <int:id>, we'll modify the request url on the client side.

Reply:
The post requires the campaign ID, that's why I have it that way

Reply:
Yea I feel we can also just pass the campaign_id in the request body as well, so if we don't need to do that way, we can